manifests/repo/debian.pp

```diff
@@ -62,4 +62,35 @@
       }
     }
   }
+
+  if ($php::globals::php_version == '7.1') {
+    # Required packages for PHP 7.1 repository
+    package { 'apt-transport-https':
```

Comment:
I'm not the original author of the module. I think it solves problems he or the users have. ;-)